deepset_cloud_sdk/s3/upload.py

```diff
@@ -146,10 +147,11 @@ async def upload_from_file(
         :return: S3UploadResult object.
         """
         async with self.semaphore:
-            with open(file_path, "rb") as file:
+            async with aiofiles.open(file_path, "rb") as file:
                 file_name = os.path.basename(file_path)
+                content = await file.read()
```

Comment:
@ArzelaAscoIi we need to do this to make this work. Not ideal but it works for now and I don't want to spend more time on it

Reply:
I think its fine :) this is run in the event loop so we shoulnt get more than 30 concurrent reads :) sounds good